src/components/backstage/plugins/Sidebar.js

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react';
+import { Title } from 'components';
+import map from 'lodash/map';
+import get from 'lodash/get';
+import formatDistanceToNow from 'date-fns/formatDistanceToNow';
+import {
+  EditOnGitHubLink,
+  NpmChip,
+  GitHubChip,
+  RoadieDocsChip,
+  GravatarImage,
+} from 'components/backstage/plugins';
+
+const DetailsListItem = ({ label, value, title }) => {
+  return (
+    <li
+      className="border-b border-gray-200 py-2 flex place-content-between"
+      title={title}
+    >
+      <span>{label}</span> <strong>{value}</strong>
+    </li>
+  );
+};
+
+const Mainatainer = ({ name, email }) => {
+  return (
+    <li>
+      <GravatarImage
+        email={email}
+        name={name}
+        alt={`The Gravatar avatar of ${name}`}
+        title={`Name: ${name}. Email: ${email}`}
+      />
+    </li>
+  );
+};
+
+async function fetchNpmDataByName ({ packageName }) {
+  const funcUrl = '/.netlify/functions/fetchNpmDataByName';
+  let response;
+  let data = {};
+
+  try {
+    response = await fetch(`${funcUrl}?packageName=${packageName}`);
+  } catch (err) {
+    console.error(err);
+    return data;
+  }
+
+  try {
+    const json = await response.json();
```

Comment:
I think you want to also check the response code is ok before attempting to return the data. Otherwise you might return some error content as a valid result:

```suggestion
    if (!response.ok) {
      return data;
    }
    
    const json = await response.json();
```

Reply:
Fixed in a future commit.